incskill_site/templates/resource4.html

```diff
@@ -309,11 +311,7 @@ <h2 id="sec-h2">Break Statement</h2>
     handy.
   </p>
   <p>
-Let's say you want to implement the same exercise as above, i.e.
-multiplying 1 by 2 and then the resulting number by 2 until it is less than
-50. Now, what if you want to stop this process as soon as the value of i×2
-is 32. Notice that there are two conditions here. To do so, the above code
-will be adapted as follows.
+    Let's say you want to implement the same exercise as above, i.e. multiplying 1 by 2 and then the resulting number by 2 until it is less than 50. Now, what if you want to stop this process as soon as the value of i×2 is 32. Notice that there are two conditions here. To do so, the above code will be adapted as follows.
```

Comment:
leave it with the newlines since its more readable in the code and there's no difference in the final html because it automatically strips extra whitespace and turns it into once space